src/dashboard/apps/core/annuaire_entreprise_api.py

```diff
@@ -0,0 +1,132 @@
+"""Dashboard core annuaire entreprise API."""
+
+import requests
+from django.conf import settings
+
+from apps.core.validators import validate_siren, validate_siret
+
+
+class CompanyInformationAdapter:
```

Comment:
I think using a [`DataClass`](https://docs.python.org/3/library/dataclasses.html) would be more explicit for each custom data and API response. WDYT?